File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ChineseMedicineClinicsSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ChineseMedicineClinicsSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- ChineseMedicineClinics SDK exists test

local sdk = require("chinese-medicine-clinics_sdk")

describe("ChineseMedicineClinicsSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
